app/models/spot.rb

```diff
@@ -48,7 +48,25 @@ def create_matches
                            .where.not(user_id: user_id)
 
     compatible_spots.each do |spot|
-      matches.create!(first_user_id: user_id, second_user_id: spot.user_id)
+      create_match(spot)
     end
   end
+
+  def create_match(spot)
+    match = matches.create!(first_user_id: user_id, second_user_id: spot.user_id)
+    notify(match, match.second_user)
+  end
+
+  def notify(match, matched_user)
+    data = {
+      name: matched_user.name,
+      avatar: matched_user.avatar.url,
+      match_id: match.id
+    }
+
+    return unless matched_user.push_token
+    NotificationService.notify([matched_user.push_token],
```

Comment:
Since you will add a initializer this call to `NotificationService` is going to change...

```
NotificationService.new([matched_user.push_token],
                                         t('api.notifications.new_match'),
                                         data)
                                         .notify
```